utility/rpm_publish/python-storops.spec

```diff
@@ -126,6 +127,20 @@ Library for managing Unity/VNX systems. Please refer to https://github.com/emc-o
 %exclude %{python3_sitelib}/storops_test*
 %endif
 
+
+%post -n python2-%{pypi_name}
+if [ "$1" -ge 1 ] ; then
+    echo NOTE: If need to managing VNX system, please install Unisphere CLI rpm package first. Contact with Dell/EMC support on that.
```

Comment:
Thanks for your comments! I updated the spec, added a new spec for storops-vnx. And the new package is uploaded here: https://github.com/emc-openstack/storops/releases/tag/r0.4.14

```
[stack@fedora25 ~]$ rpm -qp --requires /home/stack/rpmbuild/RPMS/x86_64/python2-storops-vnx-0.4.14-1.fc25.x86_64.rpm
NaviCLI-Linux-64-x86-en_US
python2-storops
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
```